src/main/scala/api/sampling/MixedProposalDistributions.scala

```diff
@@ -29,17 +29,17 @@ object MixedProposalDistributions {
 
   def mixedProposalRandom(model: StatisticalMeshModel): ProposalGeneratorWithTransition[ModelFittingParameters] = {
     val mixproposal = MixtureProposal(
-      0.5 *: RandomShapeUpdateProposal(model, 1.0, generatedBy = "RandomShape-1.0") +
+//      0.5 *: RandomShapeUpdateProposal(model, 1.0, generatedBy = "RandomShape-1.0") +
```

Comment:
why not remove commented out code? without a comment explaining why it is still there, this will not help.